### PR TITLE
fix: added spinner when loading latest or previous messages

### PR DESCRIPTION
**Pull request by The-Vector-007**

### CODE REVIEW

File: projects/commudle-admin/src/app/feature-modules/community-channels/components/discussion-community-channel/discussion-community-channel.component.ts [hunk 1]
```diff
@@ -178,8 +179,10 @@ export class DiscussionCommunityChannelComponent implements OnInit, OnChanges, O
   }
 
   getDiscussionMessages() {
+    console.log(this.loadingMessages);
```
Comment (Ashwin-op) [inline projects/commudle-admin/src/app/feature-modules/community-channels/components/discussion-community-channel/discussion-community-channel.component.ts]: Let's remove console.log

File: projects/commudle-admin/src/app/feature-modules/community-channels/components/discussion-community-channel/discussion-community-channel.component.ts [hunk 2]
```diff
@@ -178,8 +179,10 @@ export class DiscussionCommunityChannelComponent implements OnInit, OnChanges, O
...
     if ((!this.allPreviousMessagesLoaded || !this.allLatestMessagesLoaded) && !this.loadingMessages) {
       this.loadingMessages = true;
+      console.log(this.loadingMessages);
```
Comment (Ashwin-op) [inline projects/commudle-admin/src/app/feature-modules/community-channels/components/discussion-community-channel/discussion-community-channel.component.ts]: Let's remove console.log

File: projects/commudle-admin/src/app/feature-modules/community-channels/components/discussion-community-channel/discussion-community-channel.component.ts [hunk 3]
```diff
@@ -236,6 +239,7 @@ export class DiscussionCommunityChannelComponent implements OnInit, OnChanges, O
               break;
             }
           }
+          console.log(this.loadingMessages);
```
Comment (Ashwin-op) [inline projects/commudle-admin/src/app/feature-modules/community-channels/components/discussion-community-channel/discussion-community-channel.component.ts]: Let's remove console.log

**Review by Ashwin-op — CHANGES_REQUESTED** [review]

File: projects/commudle-admin/src/app/feature-modules/community-channels/components/discussion-community-channel/discussion-community-channel.component.ts (hunk 1 above)
Comment (The-Vector-007) [reply projects/commudle-admin/src/app/feature-modules/community-channels/components/discussion-community-channel/discussion-community-channel.component.ts]: removed

File: projects/commudle-admin/src/app/feature-modules/community-channels/components/discussion-community-channel/discussion-community-channel.component.ts (hunk 3 above)
Comment (The-Vector-007) [reply projects/commudle-admin/src/app/feature-modules/community-channels/components/discussion-community-channel/discussion-community-channel.component.ts]: removed

File: projects/commudle-admin/src/app/feature-modules/community-channels/components/discussion-community-channel/discussion-community-channel.component.ts (hunk 2 above)
Comment (The-Vector-007) [reply projects/commudle-admin/src/app/feature-modules/community-channels/components/discussion-community-channel/discussion-community-channel.component.ts]: removed

**Pull request merged**